src/main/scala/vectorpipe/Collate.scala

```diff
@@ -120,24 +120,24 @@ object Collate {
         (WayExtra(Seq(parent)), node.root) +: node.children.flatMap(t => work(node.root.meta.id, t))
 
       /* It's a Node that got eaten */
-      case Some(Right(p)) => Seq((NodeExtra(p, Seq(parent)), node.root))
+      case Some(Right(p)) => Seq((NodeExtra(Seq(parent)), node.root))
 
       /* Should never occur */
       case _ => sys.error("BOOM!"); Seq.empty
     }
 
     d.root match {
       /* It was a top-level Node, not part of any Way */
-      case ElementData(_, _, Some(Left(extent))) if d.children.isEmpty =>
-        compressMeta(0, GeomExtra(extent), d.root)
+      case ElementData(_, _, None) if d.children.isEmpty =>
+        compressMeta(0, NoExtra, d.root)
 
       /* It was either a Way that became a Line/Poly, or a Relation that became a MultiPoly */
-      case ElementData(_, _, Some(Left(extent))) => {
+      case ElementData(_, _, None) => {
 
         /* Combine data for Nodes which were shared by multiple Ways */
         val kids = uniqueNodes(d.children.flatMap(t => work(d.root.meta.id, t)))
 
-        ((GeomExtra(extent), d.root) +: kids)
+        ((NoExtra, d.root) +: kids)
```

Comment:
Got rid of holding lat lngs and extents in metadata; this is bloated and doesn't need to happen - geometry reconstruction needs other mechanisms to solve other use cases (like changeset geometry reconstruction).